gradle/publish.gradle

```diff
@@ -35,6 +35,9 @@ bintray {
         licenses = ['Apache-2.0']
         vcsUrl = 'https://github.com/palantir/dropwizard-web-security.git'
         publications = ['bintray']
+
+        githubRepo = 'palantir/dropwizard-web-security'
+        githubReleaseNotesFile = 'README.md'
```

Comment:
talked offline. can look here for reference: https://github.com/bintray/gradle-bintray-plugin#buildgradle